src/main/java/org/interledger/codecs/CodecContext.java

```diff
@@ -82,8 +82,8 @@ public InterledgerPacket read(final InterledgerPacketType typeId, final InputStr
   /**
    * Helper method that accepts an {@link InputStream}, detects the type of the packet to be read
    * and decodes the packet to {@link InterledgerPacket}. Because {@link InterledgerPacket} is
-   * simply a marker interface, callers might prefer to utilize the functionality supplied by {@link
-   * #readAndHandle(InputStream, Handler)} or {@link #readAndHandle(InputStream, VoidHandler)}.
+   * simply a marker interface, callers might prefer to utilize the functionality supplied by {@code
+   * #readAndHandle(InputStream, Handler)} or {@code #readAndHandle(InputStream, VoidHandler)}.
```

Comment:
These should be {@link ...} so that they can be clickable in the IDE and in any generated javadoc.